src/main/webapp/app/admin/user-management/user-management.component.html

```diff
@@ -37,11 +37,11 @@ <h4 jhiTranslate="userManagement.search.title">Filter by</h4>
                 <th jhiSortBy="email"><span jhiTranslate="userManagement.email">Email</span> <span class="fa fa-sort"></span></th>
                 <th></th>
                 <th jhiSortBy="langKey"> <span jhiTranslate="userManagement.langKey">Lang Key</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="roles.authorityName"><span jhiTranslate="userManagement.role.authorityName">Profiles</span><span class="fa fa-sort"></span></th>
-                <th jhiSortBy="lastModifiedDate"><span jhiTranslate="userManagement.role.project">Project</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="createdDate"><span jhiTranslate="userManagement.createdDate">Created Date</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="lastModifiedBy"><span jhiTranslate="userManagement.lastModifiedBy">Last Modified By</span> <span class="fa fa-sort"></span></th>
-                <th jhiSortBy="lastModifiedDate"><span jhiTranslate="userManagement.lastModifiedDate">Last Modified Date</span> <span class="fa fa-sort"></span></th>
+                <th><span jhiTranslate="userManagement.role.authorityName">Profiles</span></th>
+                <th><span jhiTranslate="userManagement.role.project">Project</span></th>
+                <th><span jhiTranslate="userManagement.createdDate">Created Date</span></th>
+                <th><span jhiTranslate="userManagement.lastModifiedBy">Last Modified By</span></th>
+                <th><span jhiTranslate="userManagement.lastModifiedDate">Last Modified Date</span></th>
```

Comment:
Sorting could not get working here?

Reply:
No. All of these don't work. Authority and Project don't work due to the select issue I talked about.
Rest of the properties have been moved to the revision entity and I couldn't find an easy way to get the sorting work as we want.